generator/uscore/uscore_generator.rb

```diff
@@ -428,6 +450,16 @@ def get_comparator_searches(search_params, sequence)
         search_code
       end
 
+      def find_comparators(search_params, sequence)
+        param_comparators = {}
+        search_params.each do |param|
```

Comment:
Get rid of lines 454 and 460, then do `search_params.each_with_object({}) do |param, param_comparators|`.

Reply:
Fixed